apps/emqx_schema_registry/src/emqx_schema_registry.erl

```diff
@@ -64,6 +67,14 @@ get_serde(SchemaName) ->
             {ok, Serde}
     end.
 
+-spec does_type_exist(schema_name()) -> boolean().
+does_type_exist(SchemaName) ->
```

Comment:
the convention is to use is_ prefix for bool check result functions.
maybe is_existing_type

Reply:
:heavy_check_mark: